contrib/inventory/foreman.py

```diff
@@ -198,6 +266,52 @@ def _get_json(self, url, ignore_errors=None, params=None):
                 break
         return results
 
+    def _use_inventory_report(self):
+        if not(self.foreman_use_reports_api):
+            return False
+        status_url = "%s/api/v2/status" % self.foreman_url
+        result = self._get_json(status_url)
+        foreman_version = (LooseVersion(result.get('version')) >= LooseVersion('1.24.0'))
+        return foreman_version
+
+    def _fetch_params(self):
+        options, params = ("no", "yes"), dict()
+        params["Organization"] = options[self.want_organization]
```

Comment:
TIL you can index a tuple with a boolean:
```
In [1]: ("no", "yes")[True]                                                                                                                                                           
Out[1]: 'yes'
```

I do wonder if we can get the report to accept real booleans since they're JSON encoded.